lucille-core/src/test/java/com/kmwllc/lucille/core/RunnerTest.java

```diff
@@ -677,15 +681,38 @@ public void testConnectorWithUnrecognizedPipeline() throws Exception {
     RunResult result =
         Runner.run(ConfigFactory.load("RunnerTest/pipelineNotFound.conf"), Runner.RunType.TEST);
     assertFalse(result.getStatus());
+
+    Collection<Thread> nonSystemThreadsAfter =
+        ThreadUtils.findThreads(t -> !ThreadUtils.getSystemThreadGroup().equals(t.getThreadGroup()));
+
+    for (Thread thread : nonSystemThreadsAfter) {
+      log.info("THREAD testConnectorWithUnrecognizedPipeline AFTER: {}", thread.getName());
```

Comment:
suggest changing all the info log statements in this test to debug, or removing them

Reply:
removed in [cdc333a](https://github.com/kmwtechnology/lucille/pull/172/commits/cdc333a75b671f998894dc5e830c75e13bb5819d)